frontend/src/components/CostTable.vue

```diff
@@ -58,9 +60,31 @@ const columns: any = [
   {
     title: "Pricing",
     key: "pricing",
-    render: (row: RowData) => {
-      return `$${row.pricing}`;
-    },
+    align: "center",
+    children: [
+      {
+        title: "Commitment",
+        key: "commitmentUSD",
```

Comment:
commitment.usd, similar below

use https://en.wikipedia.org/wiki/Reverse_domain_name_notation for keys as it dictates a consistent and hierarchical structure.